repo/fsrepo.go

```diff
@@ -493,12 +518,17 @@ func (r *FSRepo) SetAPIAddr(maddr string) error {
 	return nil
 }
 
+func APIAddrOfRoot(rootDir string) (string, error) {
```

Comment:
There were some other checks in this code you abstracted -- were they not necessary?
```
rawPath := filepath.Join(filepath.Clean(repoDir), repo.APIFile)	
		apiFilePath, err := homedir.Expand(rawPath)	
		if err != nil {	
			return "", errors.Wrap(err, fmt.Sprintf("can't resolve local repo path %s", rawPath))	
		}
```

Reply:
I think this is a bug, thanks for catching.